build.sbt

```diff
@@ -2,15 +2,15 @@ name := "ciris-aws-secretsmanager"
 organization := "com.ovoenergy"
 licenses += ("MIT", url("http://opensource.org/licenses/MIT"))
 
-scalaVersion := "3.1.0"
-crossScalaVersions := Seq(scalaVersion.value, "2.13.5", "2.12.13")
+scalaVersion := "3.2.2"
```

Comment:
Is there a reason to drop scala 3.1 support?

Reply:
Not really, happy to revert this bit.

Reply:
Also, maybe I'm confused, but I was thinking of this bit of scala docs:
```
Thanks to the guarantees of Scala 3, there is no longer a need for cross-compilation. For example, the migration from Scala 3.1 to Scala 3.2 for the large majority of the projects would only require changing the version in the build definition
```

Reply:
from https://www.scala-lang.org/blog/2022/08/17/long-term-compatibility-plans.html#:~:text=Thanks%20to%20the%20guarantees%20of,version%20in%20the%20build%20definition.

Reply:
And the latest ciris depends on scala 3.2 https://github.com/vlovgr/ciris/releases/tag/v3.1.0 , therefore we need to upgrade this as well...